app/FreshRSS.php

```diff
@@ -90,8 +90,13 @@ private static function initAuth() {
 	}
 
 	private static function initI18n() {
-		Minz_Session::_param('language', FreshRSS_Context::$user_conf->language);
-		Minz_Translate::init(FreshRSS_Context::$user_conf->language);
+		Minz_Translate::init();
```

Comment:
Is this line needed?

Reply:
Yes because if it's not initialized, there is no language to work with. I can change the behavior to have something less cumbersome if needed.